autospotting.go

```diff
@@ -3,12 +3,11 @@ package main
 import (
 	"encoding/json"
 	"flag"
-	"fmt"
 	"log"
 	"os"
 
-	autospotting "github.com/cristim/autospotting/core"
 	lambda "github.com/eawsy/aws-lambda-go/service/lambda/runtime"
+	autospotting "github.com/cristim/autospotting/core"
```

Comment:
For this, I do not know if it's a matter of editor or not, but I have a pre-commit hook checking `go fmt` on all modified files. So maybe it comes from here.